test/integration/features/story_registry.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Local image to image-registry to deployment
+
+    The user creates a local container image with an app. They then
+    push it to the openshift image-registry in their
+    project/namespace. They deploy and expose the app and check its
+    accessibility.
+
+    Scenario Outline: Start CRC
+        Given executing "crc setup" succeeds
+        When starting CRC with default bundle and hypervisor "<vm-driver>" succeeds
+        Then stdout should contain "CodeReady Containers instance is running"
+        And executing "eval $(crc oc-env)" succeeds
+        And executing "oc login --insecure-skip-tls-verify -u kubeadmin -p 9gZtH-EWZLT-mZNFP-bkKwm https://api.crc.testing:6443" succeeds
+        
+    @darwin
+        Examples:
+            | vm-driver  |
+            | virtualbox |
+            |            |
+    @linux
+        Examples:
+            | vm-driver |
+            | libvirt   |
+
+    @darwin @linux @windows
+    Scenario: Create local image
+        Given executing "git clone https://github.com/docker-library/httpd.git" succeeds
+        And executing "./httpd/update.sh" succeeds
+        And executing "cd httpd/2.4" succeeds
+        When executing "sudo podman build -t httpd:test ." succeeds
+        Then executing "sudo podman images" succeeds
+        And stdout should contain "localhost/httpd"
+        
+    @darwin @linux @windows
+    Scenario: Push local image to OpenShift image registry
+        Given executing "oc new-project testproj" succeeds
+        When executing "sudo podman login -u kubeadmin -p $(oc whoami -t) default-route-openshift-image-registry.apps-crc.testing --tls-verify=false" succeeds
+        Then stdout should contain "Login Succeeded!"
+        When executing "sudo podman push httpd:test default-route-openshift-image-registry.apps-crc.testing/testproj/httpd:test --tls-verify=false" succeeds
+
+    @darwin @linux @windows
+    Scenario: Deploy and expose the app
+        Given executing "oc adm policy add-scc-to-user anyuid -n testproj -z default" succeeds
```

Comment:
This is wrong, why you need to add anyuid scc policy for the project because the app is not able to run as random user? If that is the case we should use different image which should don't have that requirement.

Reply:
Yes, the app needs privileges. OK, I'll use a different one!

Reply:
pleae use non-root containers